src/common/runner/runner.go

```diff
@@ -284,6 +293,14 @@ func extractExternalResources(plug *plugin.Plugin, symbol string) ([]interface{}
 }
 
 func (r *Runner) isFileSkipped(p common.IParser, file string) bool {
+	if p.Name() == "Dockerfile" {
+		statFile, statFileError := os.Stat(file)
+		if statFileError != nil {
+			log.Fatal("Could not read metadata of the file for the Docker file parser.")
+		}
+		result := strings.Contains(strings.ToLower(statFile.Name()), "dockerfile")
+		return !result
+	}
```

Comment:
@sshintaku This should be implemented in the Dockerfile, in the `ValidFile` method